src/collections/_documentation/platforms/php/index.md

```diff
@@ -2,43 +2,262 @@
 title: PHP
 ---
 
-{% include learn-sdk.md platform="php" %}
+The Sentry PHP SDK provides support for PHP 7.1 or later.
 
-The [Sentry PHP SDK](https://packagist.org/packages/sentry/sentry) provides
-support for PHP 7.1 or later.
+## Integrating the SDK
 
-This documentation goes over some PHP specific things such as integrations to
-frameworks.
+Sentry captures data by using an SDK within your application’s runtime. These are platform-specific and allow Sentry to have a deep understanding of how your application works.
 
-## Integrations
+To install the PHP SDK, you need to be using Composer in your project. For more details about Composer, see the [Composer documentation](https://getcomposer.org/doc/).
+
+```bash
+composer require sentry/sdk:2.0.3
+```
+
+### Connecting the SDK to Sentry
+
+After you’ve completed setting up a project in Sentry, Sentry will give you a value which we call a DSN or Data Source Name. It looks a lot like a standard URL, but it’s just a representation of the configuration required by the Sentry SDKs. It consists of a few pieces, including the protocol, public key, the server address, and the project identifier.
+
+To capture all errors, even the one during the startup of your application, you should initialize the Sentry PHP SDK as soon as possible.
+
+```php
+Sentry\init(['dsn' => '___PUBLIC_DSN___' ]);
+```
+
+### Verifying Your Setup
+
+Great! Now that you’ve completed setting up the SDK, maybe you want to quickly test out how Sentry works. You can trigger a PHP exception by throwing one in your application:
+
+```php
+throw new Exception("My first Sentry error!");
+```
+
+## Capturing Errors
+
+In PHP you can either capture a caught exception or capture the last error with captureLastError.
+
+```php
+try {
+    $this->functionFailsForSure();
+} catch (\Throwable $exception) {
+    Sentry\captureException($exception);
+}
+```
+
+## Releases
+
+A release is a version of your code that you deploy to an environment. When you give Sentry information about your releases, you unlock many new features:
+
+- Determine the issue and regressions introduced in a new release
+- Predict which commit caused an issue and who is likely responsible (suspect commits)
+- Resolve issues by including the issue number in your commit message
+- Receive email notifications when your code gets deployed
+
+After configuring your SDK, setting up releases is a 2-step process:
```

Comment:
Add this example here:
```
```php
Sentry\init([
  'dsn' => '___PUBLIC_DSN___',
  'release' => 'your release name'
]);
```
```